### ISSUE TITLE
Could not determine java version from '9.0.4'

### ISSUE THREAD

**Issue by MysticRyuujin** [ether-camp/ethereum-harmony#56]
I'm using Ubuntu 16.04.3 x64

Not sure if Harmony error or EthereumJ error?

```
sudo add-apt-repository ppa:webupd8team/java
sudo apt install oracle-java9-installer
sudo apt install oracle-java9-set-default
```
I've also set `JAVA_HOME="/usr/lib/jvm/java-9-oracle"`
```
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine java version from '9.0.4'.
```

**Comment by zilm13** [comment]
@MysticRyuujin Looks like both are affected, we should upgrade Gradle to the latest version.

**Comment by MysticRyuujin** [comment]
I switched over to Java8 for the time being 😄 but I thought I'd report it not working on the latest.

**Comment by zilm13** [comment]
Thank you for reporting. We will fix it!

**Comment by zilm13** [comment]
Should be fixed with #66
